app/models/spotlight/page.rb

```diff
@@ -6,7 +6,7 @@ class Page < ActiveRecord::Base
     MAX_PAGES = 50
 
     extend FriendlyId
-    friendly_id :title, use: [:slugged, :scoped, :finders, :history], scope: [:exhibit, :locale]
+    friendly_id :title, use: [:slugged, :scoped, :finders, :history], scope: [:exhibit, :locale, :type]
```

Comment:
I honestly don't know why we have `friendly_id` on the base class as well as the subclasses but `/shrug`

Reply:
I think it was a problem with friendly_id + STI?

Reply:
Probably.

Reply:
the [friendly_id docs](http://norman.github.io/friendly_id/file.Guide.html#Setting_Up_FriendlyId_in_Your_Model) say:
> A note about single table inheritance (STI): you must extend FriendlyId in all classes that participate in STI, both your parent classes and their children.